rules/Symfony73/NodeAnalyzer/LocalArrayMethodCallableMatcher.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Symfony73\NodeAnalyzer;
+
+use PhpParser\Node\Expr;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Scalar\String_;
+use Webmozart\Assert\Assert;
+
+final class LocalArrayMethodCallableMatcher
+{
+    public function match(Expr $expr): ?string
+    {
+        if ($expr instanceof MethodCall) {
+            Assert::isInstanceOf($expr->name, Identifier::class);
```

Comment:
MethodCall name can be any Expr, eg variable: `$foo->$bar()` so this assert can cause error, should if instanceof Identifier instanceof instead

Reply:
Will update :+1:

Reply:
Here we go 
https://github.com/rectorphp/rector-symfony/pull/766